CMakeLists.txt

```diff
@@ -131,6 +130,10 @@ set(required_deps ASS
                   ZLIB
                   ${PLATFORM_REQUIRED_DEPS})
 
+if(NOT CORE_SYSTEM_NAME STREQUAL windows AND NOT CORE_SYSTEM_NAME STREQUAL windowsstore)
+  list(APPEND required_deps Iso9660)
+endif()
```

Comment:
no platform hack in common CMake files

Reply:
says who? There is other platform specifics in there already